addons/binding/org.openhab.binding.neeo/README.md

```diff
@@ -0,0 +1,104 @@
+1. On Windows - installing iTunes will install bonjour
+2. On Linux - please install zeroconf (see vendor documentation on how to do that)
+3. On Mac - should already be installed
+
+When you add the NEEO brain, the rooms on the brain will then be auto discovered and placed in the inbox. 
+
+
+## Thing Configuration
+
+The following configurations occur for each of the bridges/things:
+
+### NEEO Brain
+
+| Name         | Type          | Description                               |
+|--------------|---------------|-------------------------------------------|
+| ipAddress    | string        | IP Address or host name of the NEEO Brain |
+
+### NEEO Room
+
+| Name         | Type          | Description                                           |
+|--------------|---------------|-------------------------------------------------------|
+| key          | string        | The unique key identifying the room on the NEEO brain |
+
+
+## Channels
+
+### NEEO Brain
+The NEEO brain has no channel information (although it has properties you may view within the paperUI)
+
+### NEEO Room
+The NEEO Room is dynamically generated from the brain.  Each room will dynamically generate the following channel groups:
+1) Each room will have exactly one "room-state" representing the current state of the room.
+2) Each room will have zero or more "room-recipe-xxx" (where xxx is the recipe key) groups representing each recipe in the room.
+3) Each room will have zero or more "room-scenario-xxx" (where xxx is the scenario key) groups representing each scenario in the room.
+
+#### Room State Group
+The following channels will be in the room state group
+
+| Channel Type ID    | Read/Write | Item Type    | Description                                                          |
+|--------------------|------------|--------------|--------------------------------------------------------------------- |
+| activeScenarios    | R          | String       | The name of the active scenario                                      |
+| currentStep        | R          | String       | Displays the current step being executed                             |
+
+Current Step is local to what started a recipe.  If openHAB started the recipe, only openHAB will see the current step.  If 
+the NEEO Brain started the recipe, only NEEO brain will see the current step.
+
+#### Room Recipe Group
+Each room recipe group will have the following channels
+
+| Channel Type ID    | Read/Write | Item Type    | Description                                                                          |
+|--------------------|------------|--------------|------------------------------------------------------------------------------------- |
+| name               | R          | String       | The name of the recipe                                                               |
+| type               | R          | String       | The type of recipe                                                                   |
```

Comment:
What types do we have?

Reply:
No idea actually - much of this NEEO won't tell me (I don't think they want to officially support specified types).  I'll add some wording to that affect.